### PR TITLE
Fix spelling mistakes in README

### PR DESCRIPTION
**Pull request by meshpi**
I fixed various spelling mistakes with 'incognito' among other things

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Nothing Private
-This project is a proof of concept that anyone can identify and track you, even if you are using **Private Browsing** or **Icognito Mode** in your web browser. Many people thinks that they can hide their identity if they are using private browsing or icongnito, This project will prove them they are wrong.
+This project is a proof of concept that anyone can identify and track you, even if you are using **Private Browsing** or **Incognito Mode** in your web browser. Many people thinks that they can hide their identity if they are using private browsing or incognito, This project will prove them they are wrong.
```
Comment (Timothee) [inline README.md]: While you're at it, I'd fix this:

> incognito, This

Either a period or `this`.

File: README.md [hunk 2]
```diff
@@ -8,14 +8,14 @@ This project is a proof of concept that anyone can identify and track you, even
...
+Hope you are surprised! :smile:. Yes the website can remember your name even if you had visited it via the **Private browsing** or **Incognito Mode**. Yes nothing is private in this world anymore!. This is what the big companies are doing with your identity. You think that going private mode will wipe out all the traces. **Absolutely Not!**. Actually, using the private browsing or incognito mode will just help you to clear your browsing history. Your internet service providers, search engines and your favourite websites **Can still track you**. They can know you. They know what you like and dislike. They use your data to earn money. The video below explains everything:
```
Comment (Timothee) [inline README.md]: Extra period at `anymore!.` and `**Absolutely Not!**.`

Comment (Timothee) [inline README.md]: `**Can still track you**` should be `**can....`.

File: README.md [hunk 3]
```diff
@@ -26,12 +26,12 @@ Yes nothing is free...
 * TODO: more Useful blog links
 
 ### Some tech stuffs
-Nothing private uses the browser fingerprinting feature of client.js to obtain the fingerprint of your web browser. It will save the fingerprint along with your name in a Sqlite db using PHP as backend, when you submits it. Next time, when you visit the website, your browser fingerprint is matched with the cloumn in the database and it will return your name.
+Nothing private uses the browser fingerprinting feature of Client.js to obtain the fingerprint of your web browser. It will save the fingerprint along with your name in a Sqlite db using PHP as backend, when you submits it. Next time, when you visit the website, your browser fingerprint is matched with the column in the database and it will return your name.
```
Comment (Timothee) [inline README.md]: `Sqlite` -> `SQLite`.
`db` -> `DB` or `database`.
`submits` -> `submit`
`Next time, when you visit the website` -> `Next time you visit the website`

**Comment by gautamkrishnar** [comment]
Thanks a lot @meshpi for the PR. And thanks @Timothee for you review ❤️

**Pull request merged**